app/assets/v2/js/pages/bounty_details.js

```diff
@@ -1240,6 +1240,9 @@ const build_uri_for_pull_bounty_from_api = function() {
   if (typeof document.issue_stdbounties_id != 'undefined') {
     uri = uri + '&standard_bounties_id=' + document.issue_stdbounties_id;
   }
+  if (typeof document.eventTag != 'undefined') {
```

Comment:
wouldn't `if(document.eventTag)` suffice ? @octavioamu

Reply:
I think, but I believe using typeof is more secure in terms of maintainability for future changes  and following the rest of the code logic.